app/middleware/auth.js

```diff
@@ -0,0 +1,13 @@
+import firebase from "~/plugins/firebase.ts"
+export default function ({ _ }) {
+  firebase.auth().onAuthStateChanged(function (user) {
```

Comment:
```suggestion
  firebase.auth().onAuthStateChanged((user) => {
```

細かい部分ですが、es2015に従うとこちらの書き方かな？と思いました
ここは @qst-exe にも聞きたいところです

Reply:
ESLintが効いてない！
(本当は起きないはずなのでOKとしよう)